kmodes/util/dissim.py

```diff
@@ -10,6 +10,27 @@ def matching_dissim(a, b, **_):
     return np.sum(a != b, axis=1)
 
 
+def jaccard_dissim_binary(a, b, **__):
+    """Jaccard dissimilarity function for binary encoded variables"""
+    if ((a == 0) | (a == 1)).all() and ((b == 0) | (b == 1)).all():
+        return np.sum(np.bitwise_and(a, b), axis=1) / np.sum(np.bitwise_or(a, b), axis=1)
+    raise ValueError("Missing or non Binary values detected in Binary columns.")
+
+
+def jaccard_dissim_label(a, b, **__):
+    """Jaccard dissimilarity function for label encoded variables"""
+    if np.isnan(a.astype('float64')).any() or np.isnan(b.astype('float64')).any():
+        raise ValueError("Missing values detected in Numeric columns.")
+    intersect_len = np.empty(len(a), dtype=int)
+    union_len = np.empty(len(a), dtype=int)
+    i = 0
+    for row in a:
+        intersect_len[i] = len(np.intersect1d(row, b))
+        union_len[i] = len(row) + len(b) - intersect_len[i]
+        i += 1
+    return intersect_len / union_len
```

Comment:
Please handle exceptional case where `union_len == 0`.

Reply:
This should never actually happen unless both my arrays come in empty. Still, I'll put up a check, in case this situation arises.